src/server.js

```diff
@@ -18,6 +18,11 @@ var authFunction = localAuth;
 module.exports = function() {
 	manager = new ClientManager();
 
+	if (!fs.existsSync("client/js/bundle.js")) {
```

Comment:
Also, I would probably move this block into index.js. I'm not sure it belongs in server.